proxies/live/apiproxy/targets/sync-wrap.xml

```diff
@@ -123,6 +127,15 @@
         <Name>AssignMessage.Errors.CatchAllMessage</Name>
       </Step>
     </FaultRule>
+    <FaultRule name="patient_create_rate_limit_exceeded">
+      <Condition>(ratelimit.SpikeArrest.PatientCreate.failed = true)</Condition>
```

Comment:
Looking above for other examples. should this be using "Equals" instead of "=" ?

Reply:
I'll check

Reply:
you were right, have committed the fix